### PR TITLE
Improved README

### PR DESCRIPTION
**Pull request by dpobel**
* s/eZ Publish/eZ Platform/g
* Improved the paragraph on reporting bugs
* Mentioned the fact it's installed by default

### CODE REVIEW

**Comment by bdunogier** [comment]
+1

File: README.md [hunk 1]
```diff
@@ -51,13 +57,13 @@ On an existing installation, here's what you need to do:
   _ezpublishPlatformUIRoutes:
       resource: "@eZPlatformUIBundle/Resources/config/routing.yml"
   ```
-* [Configure the REST API to use the session based authentication](https://doc.ez.no/display/EZP/REST+API+Authentication).
+* Make sur [the REST API is configured to use the session based authentication](https://doc.ez.no/display/EZP/REST+API+Authentication).
```
Comment (crevillo) [inline README.md]: sure. missed e.

Comment (bdunogier) [inline README.md]: I'll fix it directly, thank you Carlos.

Comment (crevillo) [inline README.md]: :+1:

File: README.md [hunk 2]
```diff
@@ -7,21 +7,27 @@ to manage your content and administrate your eZ Platform install.
 
 If you find a bug, please create an issue [in JIRA](https://jira.ez.no/) and
 don't forget to add as much details as you can (steps to reproduce, OS and
-browser(s) versions, ...) and to put *PlatformUI* in the *Component/s* field.
-If you discover a security issue, please see how to responsibly report such
-issues on https://doc.ez.no/Security
+browser(s) versions, ...) and to put *PlatformUI (Admin UI & Content IU)* in the
```
Comment (joaoinacio) [inline README.md]: `Content IU` -> `Content UI` ?

**Pull request merged**